nomos-da/kzgrs/benches/kzg.rs

```diff
@@ -23,11 +24,15 @@ fn main() {
 // #[global_allocator]
 // static ALLOC: AllocProfiler = AllocProfiler::system();
 
-static GLOBAL_PARAMETERS: LazyLock<UniversalParams<Bls12_381>> = LazyLock::new(|| {
+static PROVING_KEY: LazyLock<UniversalParams<Bls12_381>> = LazyLock::new(|| {
+    println!("WARNING: Proving key is randomly generated. Use for development only.");
```

Comment:
Was this a println before as well? We should have a log instead I think.

Reply:
I think it's fine to have this println in benches.